src/Ripple/Ripple.js

```diff
@@ -79,9 +79,21 @@ export default class Ripple extends Component {
      * The CSS class name of the root element.
      */
     className: PropTypes.string,
+    /**
+     * TODO: ???
```

Comment:
Determines if the ripple has a pulsation animation, typically showing a keyboard focus state of an element.